setup.py

```diff
@@ -22,55 +22,6 @@
 VERSION = open('VERSION', 'r').read().strip()
 
 
-def __is_virtual_env():
-    return (hasattr(sys, 'real_prefix') or (hasattr(sys, 'base_prefix') and
```

Comment:
Looks like sys is not being used anymore, so we can remove the import.

Reply:
`sys` is used on `if sys.version[:2]`...